tensorflow/lite/g3doc/performance/post_training_quantization.md

```diff
@@ -151,6 +151,48 @@ The disadvantages of float16 quantization are as follows:
     to float32 when run on the CPU. (Note that the GPU delegate will not perform
     this dequantization, since it can operate on float16 data.)
 
+### Integer only: 16-bit activations with 8-bit weights (experimental)
```

Comment:
This looks good to me, but can we include it in https://github.com/tensorflow/tensorflow/pull/36251? I think that will make it easier to get through API review. Thanks!